app/models/mapping.rb

```diff
@@ -74,11 +74,11 @@ def display_connection
     end
   end
 
-  def get_entity
-    if entity_mappings.size > 1
-      "#{entity_mappings.first}..."
+  def get_symbol
```

Comment:
I don't see the intention of the original method being named `get_entity`, maybe @SGraef had a good reason for it.